src/themes.cpp

```diff
@@ -0,0 +1,43 @@
+#include "themes.hpp"
+#include "menu.hpp"
+
+std::string Themes::themedOutput(ull value) {
+  switch (themeCode) {
+  case 1:
+    return std::to_string(value);
+  case 2:
+    return elements[log2(value) - 1];
+  case 3:
+    return std::to_string(fibonacci_numbers[log2(value) - 1]);
+  case 4:
+    return programming_languages[log2(value) - 1];
+  default:
+    return "ERR";
+  }
+}
+
+void Themes::chooseTheme() {
+  bool err = false;
+  while ((themeCode > themeCount || themeCode < 1)) {
+    clearScreen();
+    drawAscii();
+
+    if (err) {
+      std::cout << red
+                << "  Invalid input. Theme number should range from 1 to "
+                << themeCount << "." << def;
+      endl(2);
+    }
+
+    std::cout << bold_on << "  1. Standard: 2, 4, 8, ..." << std::endl
+              << "  2. Elements: H, He, Li, ..." << std::endl
+              << "  3. Fibonacci Numbers: 1, 2, 3, ..." << std::endl
+              << "  4. Programming Languages: C, C++, PHP, ..." << std::endl
```

Comment:
Ask the question: "If I had `400` themes, how will this `game` handle "revealing the contents" in a `[theme]`?"

Currently, a `[theme]` (as this is the `[Themes]` class, `_[Themes::chooseTheme()]_`) must handle metadata of all `(potentially 4XX)` themes in the game. 
It may be a `"Theme handler / controller"` might need to be created. This `handler / controller` handles `"Themes"` inside this game such as reading and revealing snippets of data in an `[x theme]`.

This may make the code more generic so that:
* For any `[theme]` the `[handler / controller]` "looks after", get some `[data]` from a `[theme]` and make the `[game]` print it out.